### PR TITLE
Minor updates on some section with outdated information

### PR DESCRIPTION
**Pull request by mervebolatdd**
### What does this PR do? What is the motivation?

Some of the sections on Trace exploration documentation was out of date and misleading. The full list of the changes can be found here: https://docs.google.com/document/d/1Q5R119FUXj31GO86aWv5ZNFIsa7puP5WMfDCcBkmsEo/edit 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_explorer/query_syntax.md [hunk 1]
```diff
@@ -191,47 +212,14 @@ To start using an attribute as a Facet or in the search, click on it and add it
 
 {{< img src="tracing/app_analytics/search/create_facet.png" style="width:50%;" alt="Create Facet" style="width:50%;">}}
 
-Once this is done, the value of this attribute is stored **for all new traces** and can be used in [the search bar](#search-bar), [the Facet Panel](#facet-panel), and in the Trace graph query.
+Once you create a new facet, it will be available in the facet panel for quick filtering and basic analytics.
```
Comment (rtrieu) [inline content/en/tracing/trace_explorer/query_syntax.md:215]:
```suggestion
After you create a new facet, it is available in the facet panel for filtering and basic analytics.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback, but otherwise looks good!

**Review by mervebolatdd — COMMENTED** [review]
Looks great, thanks!

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**